### PR TITLE
Site updates - API and Download pages

### PR DESCRIPTION
**Pull request by arscan**
Need to update download links to point to newer samples that Jay just put together, at the very least.

Open to any recommendations for changes in language, examples, etc.

Also, I saw we have multiple full-population downloads in our downloads folder.  I referenced them as 'Version 1' and 'Version 2', as I thought it might be useful to have a version concept for our data.  But I can just reference the most recent version as well.

### CODE REVIEW

**Comment by arscan** [comment]
screen shot 2017-11-02 at 10 34 22 am
screen shot 2017-11-02 at 10 34 05 am

File: site/download.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            <div class="col-md-8 col-md-offset-1">
+              <h1 class="page-title">Download</h1>
+
+              <p>
+                 The SyntheticMass data set is available for download in bulk as gzip archives.  Each archive contains one million synthetic patient medical records,
+                 encoded in HL7 FHIR, C-CDA and CSV.  Please consider <a href="/feedback.html">reaching out</a> and letting us know how you are using this data set so we can improve it in the future.
+              </p>
+
+               <ul>
+                 <li><a href="/downloads/2017_05_24/synthea_1m_fhir_3_0_May_24.tar.gz">SyntheticMass Data, Version 2 (24 May, 2017)</a>: 21GB. FHIR 3.0.1, CSV, C-CDA</li>
```
Comment (jawalonoski) [inline site/download.html]: Add the track on click stuff...

File: site/api.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                <li><a href="/fhir/Patient/58b3663e3425def0f0f6a229">Basic information about a single patient</a></li>
+                <li><a href="/fhir/Patient/58b3663e3425def0f0f6a229/$everything">All medical record data related to a patient</a></li>
+              </ul>
+
+              <p>
+              Search Examples:
+              </p>
+
+              <ul>
+                <li><a href="/fhir/Patient?birthdate=gt1947-03-26&birthdate=lte1999-03-26&_count=100">Patients between the ages of 18 and 70</a></li>
```
Comment (jawalonoski) [inline site/api.html]: Please show the query as well, so it just isn't a link. People may want to copy and paste.

```
https://syntheticmass.mitre.org/fhir/Patient?birthdate=gt1947-03-26&birthdate=lte1999-03-26&_count=100
```

**Comment by arscan** [comment]
screen shot 2017-11-02 at 9 11 04 pm

**Comment by arscan** [comment]
Added in tracking on download link click, and now show fhir api calls on the page instead of hiding them in anchor tags.

Still need to update the example 1k downloads... haven't gotten production access sorted out yet.

Also, are you ok with the 'Version 1, Version 2' thing on the downloads (I'm not even sure if that makes sense... I just saw the files on the server and guessed that's what they represent).

**Comment by jawalonoski** [comment]
OK with versions.

**Comment by arscan** [comment]
Ok, just waiting on Denis to regrant me access to production.  When that's done I'll upload the new synthea sample files, fix those links here, merge into master, then deploy. It's up on https://syntheticmass-stg.mitre.org/ in the meantime.

**Pull request merged**